src/Toggle/Toggle.js

```diff
@@ -204,7 +209,7 @@ class Toggle extends React.Component {
     );
 
     if (this.state.switched) {
-      thumbStyles.marginLeft = `-${thumbStyles.width}`;
+      thumbStyles.marginLeft = 0 - thumbStyles.width;
```

Comment:
Actually, my suggestion won't work. It's raising a warning:
> Warning: a `div` tag (owner: `Paper`) was passed a numeric string value for CSS property `marginLeft` (value: `-20`) which will be treated as a unitless number in a future version of React.

You gonna have to use `10` instead of `10px` until we change the styling approach with the `next` branch.